createrepo_c.spec

```diff
@@ -130,79 +89,36 @@ Requires:       %{name}-libs = %{version}-%{release}
 
 %description -n python3-%{name}
 Python 3 bindings for the createrepo_c library.
-%endif
 
 %prep
 %autosetup -p1
-%if %{with python2}
-mkdir build-py2
-%endif
 
-%if %{with python3}
 mkdir build-py3
-%endif
 
 %build
-# Build createrepo_c with Python 2
-%if %{with python2}
-pushd build-py2
-  %cmake .. -DPYTHON_DESIRED:FILEPATH=%{__python2} %{!?with_zchunk:-DWITH_ZCHUNK=OFF} %{!?with_libmodulemd:-DWITH_LIBMODULEMD=OFF}
-  make %{?_smp_mflags} RPM_OPT_FLAGS="%{optflags}"
-  %if %{without python3}
-  # Build C documentation
-  make doc-c
-  %endif
-popd
-%endif
-
 # Build createrepo_c with Pyhon 3
-%if %{with python3}
 pushd build-py3
```

Comment:
The librepo PR did away with the entire pushd / popd "block", but this is structured slightly differently, so I'm not going to change that myself.

Reply:
The pushd / popd was there because it was building the whole project possibly two times, if building packages for both python 2 and python 3. So that's not needed anymore, but in the librepo PR I've also switched the build to the new rpm cmake macros and an out of tree build.

Seems make is called with arguments here and I don't know how to do that with the new macros off the top of my head, I'd need to fiddle with it. It's fine, we can clean this up later ourselves.

Reply:
`make doc-c` -> `%cmake_build --target doc-c`.

Reply:
Meant for me or for the follow-up?